shared/src/main/java/com/faforever/neroxis/util/serial/biome/TerrainMaterials.java

```diff
@@ -33,16 +21,5 @@ public class TerrainMaterials {
     private String[] normalPaths = new String[TERRAIN_NORMAL_COUNT];
     private float[] normalScales = new float[TERRAIN_NORMAL_COUNT];
     private String[] previewColors = new String[TERRAIN_NORMAL_COUNT];
-    private Integer[] terrainTypes = {
-            1,  // ground
-            1,  // groundAccent
-            1,  // plateauAccent
-            1,  // slopes
-            1,  // slopesAccent
-            40,  // beach
-            150,  // rock
-            150,  // rockAccent
-            221,  // shallowWater
-            223   // deepWater
-    };
+    private Integer[] terrainTypes = {};
```

Comment:
We should document what the expected number of terrain types is here in the initialization

Reply:
Just initialize the array as new String[TERRAIN_NORMAL_COUNT]; like the others

Reply:
Right now it is always 10, but it actually doesn't have to be. It could be any other number as well. There is no requirement for it to be 10. Does it still make sense to initialize it with a fixed length then?

Reply:
No but then the comment doesn't really make sense either.

Reply:
yeah, true. Did only think this through after I pushed

Reply:
So what do we do?

Reply:
Then you could just leave it as it was.